src/pages/resultsView/download/DownloadUtils.ts

```diff
@@ -21,12 +21,12 @@ export interface IDownloadFileRow {
 }
 
 const mrnaMap:any = {
```

Comment:
these maps are probably not necessary anymore since they map a value to itself

Reply:
Yes, I will replace them.